cmd/cue/cmd/get_go.go

```diff
@@ -530,16 +563,29 @@ func (e *extractor) extractPkg(root string, p *packages.Package) error {
 		}
 
 		file := filepath.Base(p.CompiledGoFiles[i])
-
 		file = strings.Replace(file, ".go", "_go", 1)
 		file += "_gen.cue"
+		filePath := filepath.Join(dir, file)
```

Comment:
I'd put this in an "else" for flagOutFile.IsSet below, to not give the impression that it's always used